azkaban-soloserver/src/package/conf/azkaban.properties

```diff
@@ -24,6 +24,7 @@ h2.create.tables=true
 velocity.dev.mode=false
 
 # Azkaban Jetty server properties. Ignored in tomcat
+jetty.hostname=localhost
```

Comment:
Sorry, but I didn't mean the PR title. I mean the comments in the `azkaban.properties` files. For example "# Azkaban Jetty server properties. Ignored in tomcat" above ^